### PR TITLE
v2.3 release notes

### PR DESCRIPTION
**Pull request by chensuyue**
## Type of Change

Only for review

### CODE REVIEW

File: v2.3_release_notes.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+- Improvement
+- Productivity
+- Bug Fixes
+- Examples
+- Validated Configurations
+
+
+**Highlights**
+- Supported low precision (INT4, NF4, FP4) and [Weight-Only](https://github.com/intel/neural-compressor/blob/master/docs/source/quantization_weight_only.md) quantization algorithms including RTN, AWQ, GPTQ and TEQ on ONNX Runtime and PyTorch for LLMs optimization.
+- Supported quantization for ONNX Runtime DML EP and DNNL EP.
```
Comment (hshen14) [inline v2.3_release_notes.md:15]: ONNX Runtime DML EP on Intel NPU (e.g., Meteor Lake) and DNNL EP on Intel CPU (e.g., SPR)

File: v2.3_release_notes.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- Features
+- Improvement
+- Productivity
+- Bug Fixes
+- Examples
+- Validated Configurations
+
+
+**Highlights**
+- Supported low precision (INT4, NF4, FP4) and [Weight-Only](https://github.com/intel/neural-compressor/blob/master/docs/source/quantization_weight_only.md) quantization algorithms including RTN, AWQ, GPTQ and TEQ on ONNX Runtime and PyTorch for LLMs optimization.
```
Comment (hshen14) [inline v2.3_release_notes.md:14]: Add link to AWQ, GPTQ

File: v2.3_release_notes.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+GPTQ ([b5ac3c](https://github.com/intel/neural-compressor/commit/b5ac3c4492c7f21ea0e6910eba11a637b67405f1), [6ba783](https://github.com/intel/neural-compressor/commit/6ba78372cc846ab961b73b9b7007ec41e75341e8)) and TEQ ([d2f995](https://github.com/intel/neural-compressor/commit/d2f995bf00bf808eb318887e8bbbea6e0529740e), [9ff7f0](https://github.com/intel/neural-compressor/commit/9ff7f01c3ca9f5aba0aff01260d58ce3007a8f4c)) for PyTorch.   
+- [Quantization] Support NF4 and FP4 data type for PyTorch Weight-Only ([3d11b5](https://github.com/intel/neural-compressor/commit/3d11b5e78d7bddcdee56f354de9d1f78a3da2033))
```
Comment (hshen14) [inline v2.3_release_notes.md:23]: Weight-Only -> Weight-Only quantization

File: v2.3_release_notes.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+- Highlights
+- Features
+- Improvement
+- Productivity
+- Bug Fixes
+- Examples
+- Validated Configurations
+
+
+**Highlights**
```
Comment (hshen14) [inline v2.3_release_notes.md:13]: Add Sparsity (sparseGPT) as 3rd highlight

Comment (hshen14) [reply v2.3_release_notes.md:13]: Add INC integration into ONNX Runtime and Olive as the 1st highlight

**Comment by hshen14** [comment]
Weight-only vs. Weight-Only, please unify

File: v2.3_release_notes.md (hunk 4 above)
Comment (chensuyue) [reply v2.3_release_notes.md:13]: updated

**Review by hshen14 — APPROVED** [review]

**Pull request closed without merging**